src/openrct2-ui/input/keyboard_shortcut.c

```diff
@@ -278,6 +278,27 @@ static void shortcut_invisible_people_toggle()
     toggle_view_flag(VIEWPORT_FLAG_INVISIBLE_PEEPS);
 }
 
+static void shortcut_gridlines_toggle()
+{
+    if (gScreenFlags & SCREEN_FLAGS_TITLE_DEMO)
+        return;
+
+    toggle_view_flag(VIEWPORT_FLAG_GRIDLINES);
+    
+    /* For now, the code acts as a simple toggle for the gridline display, but
+     * a full implementation of the feature will require the option be saved to
+     * disk depending on which way the keypress toggled the display. This
+     * interacts somewhat complexly with both the various construction modes as
+     * well as the checkbox for gridlines display in Options.
+     *
+     * The following section serves as a rudimentary implementation of this
+     * functionality. Before uncommenting these lines, please move or copy your
+     * hotkeys.dat file to a new location.
+    gConfigGeneral.always_show_gridlines ^= 1;
+    config_save_default();
```

Comment:
Please take out this comment. It's better to rename the option in the Options window to 'Always show gridlines on landscape'. After all, if someone wants permanent gridlines, they'd use that option, and if they want it temporarily, they'd use this shortcut.